third_party/protobuf

```diff
@@ -1 +1 @@
-Subproject commit bdeb215cab2985195325fcd5e70c3fa751f46e0f
+Subproject commit 4b36284bdfdc77896d7c4cb9645433cf86efb069
```

Comment:
it looks like this commit does not exist at all in google/protobuf repository.

v3.0.0-beta-4 seems to be commit 56855f6.
https://github.com/google/protobuf/releases